TaffoInitializer/IndirectCallPatcher.cpp

```diff
@@ -0,0 +1,174 @@
+#include "IndirectCallPatcher.h"
+
+#include "Metadata.h"
+#include "TaffoInitializerPass.h"
+#include "TypeUtils.h"
+#include "llvm/ADT/Statistic.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Debug.h"
+#include <llvm/IR/IRBuilder.h>
+#include <llvm/Transforms/Utils/Cloning.h>
+#include <llvm/Transforms/Utils/ValueMapper.h>
+
+using namespace taffo;
+using namespace llvm;
+
+/// Check recursively whether an unsupported function is called.
+bool containsUnsupportedFunctions(llvm::Function *function, int exploreDepth) {
+  std::vector<std::string> prefixBlocklist{"__kmpc_omp_task"};
```

Comment:
Why local? Should it be a global or static const?

Reply:
I was unsure on whether to have it local or global since it's likely going to be used only from that function, but maybe it's more clear to make it global? 
Even if the constant is not included in the header so it wouldn't mess with the namespace if global, I thought that it was specific to the `containsUnsupportedFunctions` check.

Agree on having it as a static const.

Reply:
Since you're using it only here, makes sense to keep it local. However, you should be careful with the local variables you allocate in a recursive function. I would prefer having it static, so you only allocate it once.